alf/bin/train.py

```diff
@@ -180,6 +180,11 @@ def main(_):
                 rank=0, world_size=1, conf_file=conf_file, root_dir=root_dir)
             return
 
+        # The other process will communicate with the authoritative
+        # process via network protocol on localhost:12355.
+        os.environ['MASTER_ADDR'] = 'localhost'
```

Comment:
Is it better to add prefix to these env variables, e.g., "ALF_"?

Reply:
I do agree. Sadly this is hard-coded in pytorch. This is a pytroch distributed protocol.